cmake/schemes/url_sha1_openssl_ios.cmake.in

```diff
@@ -131,6 +138,12 @@ foreach(variant @IPHONEOS_ARCHS@ @IPHONESIMULATOR_ARCHS@)
       make install_sw
           # installed libraries will be rewrited,
           # other files are equal
+      COMMAND # Copy license files
+      "@CMAKE_COMMAND@"
+      "-Dsrcdir=@HUNTER_PACKAGE_SOURCE_DIR@"
+      "-Ddstfile=@HUNTER_PACKAGE_LICENSE_FILE@"
+      -P
+      "@HUNTER_SELF@/scripts/try-copy-license.cmake"
```

Comment:
Only one command needed. This one can be removed since we already have this step in `"@HUNTER_EP_NAME@-ios_universal"`